cli/deposit.py

```diff
@@ -54,12 +54,12 @@ def check_python_version():
     '--num_validators',
     prompt='Please choose how many validators you wish to run',
     required=True,
-    type=int,
+    type=int,  # type: ignore
```

Comment:
There was a mypy error `error: Function is missing a return type annotation` that  I don't understand why. Ignoring it now.